token-lending/cli/src/main.rs

```diff
@@ -612,6 +625,14 @@ fn main() {
             let flash_loan_fee = value_of::<f64>(arg_matches, "flash_loan_fee");
             let host_fee_percentage = value_of(arg_matches, "host_fee_percentage");
             let deposit_limit = value_of(arg_matches, "deposit_limit");
+            let borrows_disabled_option = value_of(arg_matches, "borrows_disabled");
+            let borrows_disabled = match borrows_disabled_option {
+                Some(v) => match v {
```

Comment:
can't just do like `unwrap_or(true)?`

Reply:
Nope. A missing value is different from false. I made the CLI so that you don't have to specify every value every time you update the reserve config, you only specify the ones that you want to change. That makes it so that None, is different from False.